sdk/src/Core/Amazon.Runtime/ClientConfig.cs

```diff
@@ -720,33 +711,6 @@ protected ClientConfig(IDefaultConfigurationProvider defaultConfigurationProvide
             Initialize();
         }
 
-        public ClientConfig() : this(new LegacyOnlyDefaultConfigurationProvider())
```

Comment:
This constructor was not used by the service clients; I did have to modify one test to pass in a mocked provider.